ni_measurementlink_generator/ni_measurementlink_generator/template.py

```diff
@@ -104,6 +105,22 @@ def _resolve_service_class(service_class: str, display_name: str) -> str:
     "--directory-out",
     help="Output directory for measurement files. Default: '<current_directory>/<display_name>'",
 )
+@click.option(
+    "-a",
+    "--annotations-description",
```

Comment:
"annotations" is an implementation detail. As far as users are concerned, this is "the description".

I recommend renaming this to `--description` and moving it above `--description-url` in both the options and all parameter lists. 

Unfortunately, we wasted `-d` on `--description-url`, so we can't use it for `--description`. You could use `-D`, though.

BTW, if we were leaving `--annotations-description` as it is, I would prefer not using a short option rather than wasting `-a` on something that no one will remember is connected to the letter "A".

Reply:
Fixed